### ISSUE TITLE
accessing family contact info

### ISSUE THREAD

**Issue by chicklegirl** [jamestunnell/homeschool-co-op#8]
We need a way for co-op admins to access family contact info, so we can get in touch with parents, including when we need to let them know the total amount owed for tuition and registration.

**Comment by jamestunnell** [comment]
It might be easier on whoever's handling registration if I set up an automatic or semiautomatic email to inform each parent about how much they owe. This would eliminate a lot of tedious work on their part.

What other contact situations did you have in mind?

**Comment by chicklegirl** [comment]
We need to be able to contact parents to volunteer to help with duties around the co-op, as well as emergency contact--since we no longer have folks filling out paper forms.

**Comment by jamestunnell** [comment]
I have a prototype running that will provide basic contact functionality. It involves two aspects:
1. Contact lists that can be accessed from the account page. Different contact lists are made available. 
2. A contact form that will direct messages to other users. These forms are generated from a "Send Message" button that is displayed in the contact list.

You can see this feature in action (as it currently stands) in [this screencast](https://www.dropbox.com/s/xwq150bas7sg9nt/contacts_feature.mp4?dl=0)

I would appreciate any feedback you have on this feature.

**Comment by chicklegirl** [comment]
Yes, this looks like it could work for our needs. Also, I just realized, I need a way to access parents' email addresses so I can send them a statement of how much they owe for registration and tuition. The term starts tomorrow, and I've been unable to email these reminders to parents. How soon could these features be implemented, so I can access those email addresses?

**Comment by chicklegirl** [comment]
P.S. Sorry to take so long to get back to you about this. I've been out of town.

**Comment by jamestunnell** [comment]
The feature I demo'd is ready to deploy. It doesn't display the email address, but when you send a message it will go to the right address. Is that sufficient or do you need to see the email address? Or did you mean that you need to see at at-a-glance how much they owe?

**Comment by chicklegirl** [comment]
In the past, I believe the person in charge of registration calculated a breakdown of what each family owed, then sent individual emails with that information and a reminder about the first day of class. 

I'm sensing you are resistant to allow us to have access to this information. Are you concerned about protecting privacy? Because in the past, all this information was collected on a paper form, which was then kept on file, so that the planning committee could contact parents as needed. So this would be the same level of privacy as before. I believe that the members of the planning committee appreciate the sensitive nature of the information and are committed to confidentiality. 

Sent from Yahoo Mail on Android

**Comment by jamestunnell** [comment]
No, I'm not trying to be resistant. But I have a working feature already, and I just need to know if you want that to be changed. I think it's probably fine for the admins to see the email address if they need to, I just didn't know if they would need to see it if a "Send Message" button was going to be provided. It's your call.

**Comment by chicklegirl** [comment]
James, our planning committee really appreciates all your hard work on the website and the time and attention you're giving to this project. The whole site looks fantastic and has been designed in such a way that we've been able to learn to navigate it easily.

While we like the idea of a contact feature, what we need for registration purposes is access to a raw report we can generate as needed that includes phone, email, enrollments, and students for each family. Is there any way to accommodate the generation with this type of information?

```
 On Tuesday, October 6, 2015 7:02 PM, James Tunnell <notifications@github.com> wrote:
```

 No, I'm not trying to be resistant. But I have a working feature already, and I just need to know if you want that to be changed. I think it's probably fine for the admins to see the email address if they need to, I just didn't know if they would need to see it if a "Send Message" button was going to be provided. It's your call.—

**Comment by jamestunnell** [comment]
Sure, I think I understand what you have in mind.

**Comment by jamestunnell** [comment]
How does this look?
![enrollments_admin_view](https://cloud.githubusercontent.com/assets/2146252/10387108/46c7d1c6-6e13-11e5-80ab-867f7ec22ae3.png)

**Comment by chicklegirl** [comment]
Thanks for your patience in waiting for me to get back to you. My kids have been sick and we even had an ER visit over the weekend. Fortunately, everyone is much better.
Yes, this is exactly what we needed--thank you so much! It looks just right.

```
 On Thursday, October 8, 2015 11:21 PM, James Tunnell <notifications@github.com> wrote:
```

 How does this look?

**Comment by jamestunnell** [comment]
Resolved with commit a93d441
